Themes/default/css/index.css

```diff
@@ -710,345 +898,705 @@ img.sort, .sort {
+	position: absolute;
+	text-shadow: var(--topmenu-text-shadow);
+	text-transform: var(--topmenu-text-transform);
+	top: 100%;
+	width: var(--topmenu-width);
+	z-index: 2;
 }
-
-#main_menu {
-	background: var(--menu_bg_color);
-	border-top: var(--menu_border_width) var(--menu_border_style) var(--menu_border_color);
-	border-bottom: var(--menu_border_width) var(--menu_border_style) var(--menu_border_color);
+.top_menu:hover {
+	background: var(--topmenu-bg_hover);
+	border-color: var(--topmenu-border-color_hover);
+	border-width: var(--topmenu-border-width_hover);
+	box-shadow: var(--topmenu-box-shadow_hover);
+	color: var(--topmenu-color_hover);
+}
+.top_menu:is(:focus, :focus-within) {
+	background: var(--topmenu-bg_focus);
+	border-color: var(--topmenu-border-color_focus);
+	border-width: var(--topmenu-border-width_focus);
+	box-shadow: var(--topmenu-box-shadow_focus);
+	color: var(--topmenu-color_focus);
+}
+.top_menu.visible {
+	display: block;
 }
-@media (min-width: 720px) {
-	#mobile_user_menu .popup_window {
-		display: flex;
-		justify-content: space-between;
+@media (hover) {
+	.top_menu.scrollable {
+		max-height: var(--topmenu-height);
+		overflow: auto;
 	}
 }
 /* User information. */
-#profile_menu_top::after {
-	content: " \25BC";
+.top_menu .header {
+	background: var(--topmenu-header-bg);
+	border-color: var(--topmenu-header-border-color);
+	border-style: var(--topmenu-header-border-style);
+	border-width: var(--topmenu-header-border-width);
+	font-size: var(--topmenu-header-font-size);
+	padding-block: var(--topmenu-header-padding-block);
+	padding-inline: var(--topmenu-header-padding-inline);
+}
+.top_menu .body {
+	background: var(--topmenu-body-bg);
+	border-color: var(--topmenu-body-border-color);
+	border-style: var(--topmenu-body-border-style);
+	border-width: var(--topmenu-body-border-width);
+	font-size: var(--topmenu-body-font-size);
+	padding-block: var(--topmenu-body-padding-block);
+	padding-inline: var(--topmenu-body-padding-inline);
+}
+#profile_menu .header {
+	align-items: center;
+	display: grid;
+	gap: var(--topmenu-header-padding-inline);
+	grid-template-columns: auto 1fr;
 }
-.profile_user_info, .profile_user_avatar {
-	display: inline-block;
-	margin: 0 9px;
-	vertical-align: middle;
+#profile_menu time {
+	font-size: 0.65em;
+	padding-block: 0.25em;
+}
+#profile_menu a:has(img.avatar) {
+	display: flex;
+}
+#profile_menu img.avatar {
+	height: 3em;
+	width: 3em;
 }
-.profile_user_avatar img.avatar {
-	max-width: 100px;
+.profile_user_info {
+	display: grid;
+	gap: 0.25em;
 }
-.profile_username {
-	font-size: var(--profile_username_font_size);
+#profile_menu .profile_username {
+	font-size: 1.1em;
 	display: block;
-	margin-bottom: 3px;
 }
-.profile_user_links ol {
-	margin-left: 0;
-	clear: both;
-	column-count: 2;
+#profile_menu .profile_group {
+	font-size: 0.7em;
+}
+ol.profile_user_links {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	padding-block: 0.25em;
+	padding-inline: 0.25em;
```

Comment:
Is there any advantage to this new syntax if both or even all four of the values are the same?

Reply:
Not really, I probably intended to add variables to the list itself too, I might do that...